src/uct/tcp/tcp_cm.c

```diff
@@ -748,6 +749,11 @@ ucs_status_t uct_tcp_cm_handle_incoming_conn(uct_tcp_iface_t *iface,
         return UCS_ERR_UNREACHABLE;
     }
 
+    status = uct_tcp_iface_set_sockopt(iface, fd, 1);
```

Comment:
why not iface->config.conn_nb?

Reply:
`conn_nb` means that we want to have a non-blocking `connect()` call
here we are handling `fd` from `accept()`

Reply:
ok